test/babelish/test_csv2android.rb

```diff
@@ -24,15 +27,36 @@ def test_converting_csv_with_special_chars
 
   def test_converting_csv_to_dotstrings_one_output_option
     csv_file = "test/data/test_data.csv"
-    single_file = 'myApp.xml'
-    converter = Babelish::CSV2Android.new(csv_file,
-    {'English' => "en"},
-    :output_basename => 'myApp',
-    :ignore_lang_path => true)
+    single_file = "myApp.xml"
+    converter = Babelish::CSV2Android.new(
+      csv_file,
+      { "English" => "en" },
+      :output_basename => "myApp",
+      :ignore_lang_path => true
+    )
     converter.convert
     assert File.exist?(single_file), "the ouptut file does not exist"
 
     # clean up
     system("rm -rf ./" + single_file)
   end
+
+  def test_converting_with_comments
+    csv_file = "test/data/test_data_with_comments.csv"
+    french_file = "values-fr/strings.xml"
+    expected_output = File.read("test/data/test_data_fr_with_comments.xml")
+    converter = Babelish::CSV2Android.new(
+      csv_file,
+	  { "French" => "fr" },
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.